src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskForStreamingIT.java

```diff
@@ -41,109 +41,114 @@
  * com.snowflake.kafka.connector.internal.streaming.SnowflakeSinkServiceV2}
  */
 public class SnowflakeSinkTaskForStreamingIT {
-
+  private int taskId;
+  private int partitionCount;
   private String topicName;
-  private static int partition = 0;
-  private TopicPartition topicPartition;
   private Map<String, String> config;
-
-  @Mock Logger logger = Mockito.mock(Logger.class);
-
-  @InjectMocks @Spy
-  private LoggerHandler loggerHandler = Mockito.spy(new LoggerHandler(this.getClass().getName()));
-
-  @InjectMocks private SnowflakeSinkTask sinkTask1 = new SnowflakeSinkTask();
-
+  private List<TopicPartition> topicPartitions;
+  private SnowflakeSinkTask sinkTask;
+  private InMemorySinkTaskContext sinkTaskContext;
+  private List<SinkRecord> records;
+  private Map<TopicPartition, OffsetAndMetadata> offsetMap;
+
+  // sets up default objects required for sunny day tests
```

Comment:
is there some hidden logic behind this phrase `sunny day tests?`

Reply:
oh oops, I've always used 'sunny day tests' as tests that test the ideal state of the code. Ignoring exceptions or anything like that. I'll change the comment, might just be a msft term

sunny day -> ideal test cases
rainy day -> exception or breaking test cases

Reply:
Got it, I thought you are `mocking` seattle weather here!